code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -297,6 +319,17 @@ USE THIS CHEMISTRY DISPENSER FOR MAPS SO THEY START AT 100 ENERGY
 	name = "Booze Dispenser"
 	icon_state = "booze_dispenser"
 	dispensable_reagents = list("beer", "whiskey", "tequila", "vodka", "vermouth", "rum", "cognac", "wine", "kahlua", "ale", "ice", "water", "gin", "sodawater", "cola", "cream","tomatojuice","orangejuice","limejuice","tonic")
+	component_parts = newlist(
++		/obj/item/weapon/circuitboard/chem_dispenser/booze_dispenser,

```

Comment:
It totally wasn't because I copy pasted it after choosing the wrong branch and didn't want to redo it.

That would make it
New(
componet_parts = newlist(
...
)
)
Right?